assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -68,6 +69,14 @@ function MultiSelectionMovable({ selectedElements }) {
   useGlobalKeyDownEffect('meta', () => setCanSnap(false), [setCanSnap]);
   useGlobalKeyUpEffect('meta', () => setCanSnap(true), [setCanSnap]);
 
+  // ⇧ key rotates the element 30 degrees at a time
+  useGlobalKeyDownEffect('shift', () => setThrottleRotation(true), [
+    setThrottleRotation,
+  ]);
+  useGlobalKeyUpEffect('shift', () => setThrottleRotation(false), [
+    setThrottleRotation,
+  ]);
+
```

Comment:
Move this and `useState` to custom hook to be reused? Also, maybe it should just be called `isShiftPressed` or something similarly generic?

Reply:
It could even be an even more generic hook also to be used for `meta` right above:

```js
  const isMetaPressed = useIsKeyPressed('meta');
  const isShiftPressed = useIsKeyPressed('shift');
```

Reply:
This requires the keyboard to have a context provider, a bit bigger than the scope of this PR. I filed https://github.com/google/web-stories-wp/issues/910 for it